extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/dialect/QuarkusPostgreSQL10Dialect.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.hibernate.orm.runtime.dialect;
+
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+import org.hibernate.dialect.PostgreSQL95Dialect;
+import org.hibernate.engine.jdbc.env.spi.IdentifierCaseStrategy;
+import org.hibernate.engine.jdbc.env.spi.IdentifierHelper;
+import org.hibernate.engine.jdbc.env.spi.IdentifierHelperBuilder;
+
+/**
+ * Subclass of PostgreSQL10Dialect fixing schema updates by considering unquoted identifiers as lower case.
+ * This is PostgreSQL's behavior.
+ * See https://github.com/quarkusio/quarkus/issues/1886
+ */
+public class QuarkusPostgreSQL10Dialect extends PostgreSQL95Dialect {
```

Comment:
What's this one about? A move of class or a net new feature?

Reply:
Ah silly me, it's supposed to extend `PostgreSQL10Dialect`. Let me fix that.

Reply:
Fixed.